src/current/_includes/v23.1/start-in-docker/mac-linux-steps-single-node.md

```diff
@@ -105,21 +103,25 @@ The `cockroach` process listens on `127.0.0.1:26257` and `localhost:26257`, and
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    docker log roach-single --follow
+    docker logs --follow roach-single
     ~~~
 
-    To stop monitoring the logs, press Ctrl+C to exit the `docker log` command.
+    To stop monitoring the logs, press Ctrl+C to exit the `docker logs` command.
 
-1. To connect to the cluster interactively using the `cockroach sql` command-line interface, set `--url` cluster's SQL connection string, which is printed next to `sql:` in the cluster's startup details. You can replace the IP address with the hostname of the Docker container so that the script continues to work if the IP address changes. To run `cockroach sql` on the `roach-single` cluster:
+1. To connect to the cluster interactively using the `cockroach sql` command-line interface, set `--url` cluster's SQL connection string, which is printed next to `sql:` in the cluster's startup details. To run `cockroach sql` on the `roach-single` cluster:
```

Comment:
```suggestion
1. To connect to the cluster interactively using the `cockroach sql` command-line interface, set `--url` cluster's SQL connection string, which is printed next to `sql:` in the cluster's startup details. Connect to the `roach-single` cluster:
```

Reply:
I'm not sure this suggestion got applied -- maybe you haven't pushed your latest changes yet.

Reply:
Maybe I just needed to refresh.

Reply:
Sorry! I didn't use the UI to implement your suggestions. I implemented them as part of https://github.com/cockroachdb/docs/pull/18045/commits/12566040e7d99670bda1187c0f985dbd71affcca.